src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -182,9 +217,42 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
         );
     };
 
+    const handlePopupConfirmation = () => {
+        // This should reset all the forms' contents
```

Comment:
then maybe rename this function to something like `ResetForm` ?

Reply:
I created a new function to reset the form, code is clearer now